lib/assesments/assessment_2/fi_as2_saleh/screens/home/fi_as2_saleh_home.dart

```diff
@@ -0,0 +1,233 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_as2_saleh/constants/constants.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_as2_saleh/screens/profile/fi_as2_saleh_profile.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi_as2_saleh/screens/home/widgets/fi_as2_saleh_avatar.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class Home extends StatefulWidget {
+  const Home({super.key});
+
+  @override
+  State<Home> createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+
+  List<String> stories = [
+    "assets/girl1.png",
+    "assets/man1.png",
+    "assets/girl2.png",
+    "assets/man2.png",
+  ];
+
+  final Widget smallPadding = const SizedBox(
```

Comment:
prefer const here as this is a widget, 
```const Widget smallSpace = SizedBox();```

Also this isn't a padding

Reply:
can't declare as const without static
![image](https://github.com/dexplat/flutter-interns-summers-23/assets/60255252/3634378b-4260-4ace-b371-7cfc4f1031d5)